easybuild/easyconfigs/c/Circos/Circos-0.69-6-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'Circos'
+version = '0.69-6'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.circos.ca/'
+description = """Circos is a software package for visualizing data and information.
+ It visualizes data in a circular layout - this makes Circos ideal for exploring
+ relationships between objects or positions."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
@SethosII The most recent easyconfig for `Circos` uses the `GCCcore` subtoolchain rather than the full `foss` toolchain.

Any particular reason you're not doing that here? Based on the description, it doesn't seem like `Circos` is performance-sensitive?